README.md

```diff
@@ -98,11 +87,6 @@ It is also possible to activate a workspace from a subdir of a workspace root. I
 source ww .
 ```
 
-The `<workspace_name>` will be activated without navigating to the workspace root folder. The cwd will still be at `<workspace_name>/Projects/myproject`.
```

Comment:
Any reason to remove this docs?

Reply:
I added a second line in the Linux command example. Check if it fits.

Reply:
Ok. This sounds good. Changing cwd is actually an unexpected behavior for first time users.